### PR TITLE
Fix for REGISTRY-3751

### PR DESCRIPTION
**Pull request by madawas**
Fixing sorting issue

### CODE REVIEW

File: apps/store/themes/store/js/categorization.js [hunk 1]
```diff
@@ -17,39 +17,62 @@
...
                         queryObjValue.push(obj.split(":")[1].split('\"').join(''));
                     }
                 }
                 queryObj.queryValue = queryObjValue;
                 queryObjArray.push(queryObj);
             }
 
-            for(var i in queryObjArray){
-                $('.categorization-checkbox:checkbox').each(function() {
+            for (var i in queryObjArray) {
```
Comment (mushthaq33) [inline apps/store/themes/store/js/categorization.js]: Isn't it better to have a meaningful object name rather that having it as "i"

Comment (madawas) [inline apps/store/themes/store/js/categorization.js]: @mushthaq33 this code block is replaced by 6fe1ed65758f492c9d025ff984bd593e35c04530

File: apps/store/themes/store/js/categorization.js [hunk 2]
```diff
@@ -17,39 +17,62 @@
...
+    var getParameterByName = function (name, url) {
+        if (!url) {
+            url = window.location.href;
+        }
+        var regex = new RegExp("[?&]" + name + "(=([^&#]*)|&|#|$)");
+        var results = regex.exec(url);
+        if (results && results[2]) {
+            return decodeURIComponent(results[2].replace(/\+/g, " "));
+        }
+
```
Comment (sinthuja) [inline apps/store/themes/store/js/categorization.js]: Remove unnecessary new lines.

Comment (madawas) [inline apps/store/themes/store/js/categorization.js]: @sinthuja Thanks for pointing out. This line has been already removed by 0265b89d3db44295f85544d8d209d5cdbc39d56d. Therefore the extra new line does not exist in the latest version of this file.

**Pull request merged**